packages/waffle-chart/src/hooks/useWaffleChart.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+export const useWaffleChart = config => {
+  // TODO: move these to state?
```

Comment:
Seems like maybe this could be a reducer? I'm not really sure I understand why this is needed.